File: app/src/main/java/com/p1/uber/include/MyToolbar.java
package com.p1.uber.include;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.p1.uber.R;

public class MyToolbar {

     public static void show(AppCompatActivity activity, String title, boolean upButton){
            Toolbar toolbar= activity.findViewById(R.id.toolbar);
         activity.setSupportActionBar(toolbar);
         activity.getSupportActionBar().setTitle(title);
         activity.getSupportActionBar().setDisplayHomeAsUpEnabled(upButton);

     }
}

File: app/src/main/java/com/p1/uberfares/include/MyToolbar.kt
package com.p1.uberfares.include

import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.p1.uberfares.R


object MyToolbar {

    fun show(activity: AppCompatActivity, title: String?) {
        val toolbar = activity.findViewById<Toolbar>(R.id.toolbar)
        activity.setSupportActionBar(toolbar)
        activity.supportActionBar!!.title = title
        activity.supportActionBar!!.setDisplayHomeAsUpEnabled(true)

        toolbar.setNavigationIcon(R.drawable.pngegg)

    }
}